main.tf

```diff
@@ -36,7 +35,7 @@ module "object_storage" {
   source = "./modules/object_storage"
 
   namespace = var.namespace
-  labels    = local.labels
+  labels    = var.labels
```

Comment:
Note for posterity: we need to ensure that all submodules either accept labels or don't need to.

Reply:
I double-checked and found one that I missed! Redis can receive labels, but the rest are good to go. I just added it.